src/style.css

```diff
@@ -1,3 +1,147 @@
-body {
+html body {
+  font-family: Roboto, Arial, sans-serif;
+  --header-menu-active-color: rgba(0, 172, 220, 1);
+  --menu-apple-width: 37px;
+  --blue-underline-height: 4px;
+  --blue-underline-redius: 30%;
+  --total-length-horizontal-margins-inside-header: 140px;
```

Comment:
Most of this variables are unnecessary. Dont create variables for values that are used only once.